WordPress/Classes/ViewRelated/Blog/WPStyleGuide+Blog.swift

```diff
@@ -19,20 +19,17 @@ extension WPStyleGuide {
     }
 
     @objc public class func configureCellForLogin(_ cell: WPBlogTableViewCell) {
-        // TODO: make this dynamic size once @elibud's dynamic type code is merged
-        cell.textLabel?.font = WPFontManager.systemSemiBoldFont(ofSize: 15.0)
-        cell.textLabel?.sizeToFit()
         cell.textLabel?.textColor = .text
+        cell.textLabel?.font = WPStyleGuide.fontForTextStyle(.subheadline, fontWeight: .medium)
```

Comment:
Minor: since this is already a class func, we don't strictly need to reference the Type to access the static property (could just be `fontForTextStyle` instead of `WPStyleGuide.fontForTextStyle`

Reply:
Actually, it does need `WPStyleGuide`. Otherwise it defaults to expecting `UIFont` and fails.
`Type 'UIFont?' has no member 'fontForTextStyle'`

Reply:
I'm not sure what I am not seeing, but if I do this
`cell.textLabel?.font = fontForTextStyle(.subheadline, fontWeight: .medium)`

it builds fine

Reply:
🤦‍♂ I'm a dork. I left the `.`. On it!